values-vi/strings.xml

```diff
@@ -2484,7 +2484,7 @@ Bởi việc kết nối trò chơi với tài khoản diễn đàn TheoTown, b
     <string name="settings_verticaltoolbar">Thanh công cụ dọc</string>
     <string name="settings_darktoolbar">Thanh công cụ tối</string>
     <string name="dialog_tutwelcome_title">Chào mừng đến Theotown</string>
-    <string name="dialog_tutwelcome_text">Chúng tôi sẽ bắt đầu hướng dẫn bạn</string>
+    <string name="dialog_tutwelcome_text">Chúng tôi sẽ bắt đầu hướng dẫn bạn. Bạn có thể tham gia group facebook Theotown Việt Nam để giao lưu với mọi người.</string>
```

Comment:
Another advertisment

Reply:
Sorry if Vietnamese translation is not the right place to advertise Theotown Vietnam community.
I just think it would be great for Theotown community to have groups where Theotown players can share, inspire and learn from other players. 
In Vietnam, Facebook is the biggest social network, so I think it would be good if Theotown Vietnam facebook group can spread to more people. That helps new members to learn from the experience of old players and quickly progress, the community will also stick more and love Theotown more.
In that group, we also regularly organize Theotown advertising campaigns, helping more Vietnamese people to know about Theotown.

Reply:
Regardless, that's not to be allowed.

Advertising these groups in game strings would imply that they are official groups supported by us, which is not true.

Moreover, what's to say that I can't advertise my own group in Lithuanian translations and so on. It becomes a slippery slope where people will look at this, think that such advertising is okay, and edit the strings to do so, additionally hurting the player and the integrity of the translation meaning.